conf/messages.en

```diff
@@ -427,7 +427,14 @@ cds.subscription.middle-name.error.too-long=The middle name must be 35 character
 cds.subscription.last-name.error.empty=Enter your last name
 cds.subscription.family-name.error.empty=Enter your family name
 cds.subscription.last-name.error.wrong-format=Enter a last name without invalid characters
+
 cds.subscription.name-dob-details.page-error.date-of-birth=Enter your date of birth
+date-of-birth.day.empty=Date of birth must include a day
+date-of-birth.month.empty=Date of birth must include a month
+date-of-birth.year.empty=Date of birth must include a year
+date-of-birth.day-date-of-birth.month.empty=Date of birth must include a day and month
```

Comment:
messages format matches that of the field key, this means the code can be generic rather than having to specify exactly which form we are validating for